lib/index.js

```diff
@@ -0,0 +1,2 @@
+require("babel/register");
```

Comment:
Consider this comment redacted. Looks like this was fixed in Babel

https://github.com/babel/babel/blob/91fb64eaf53c67a7ed7605aab71f8dcaa4d39ae2/packages/babel/src/api/register/node.js#L115